website/app/components/doc/image-caption/index.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="doc-image-caption doc-text-body-small" ...attributes>
+  {{#if @text}}
+    {{@text}}
+  {{/if}}
```

Comment:
If you think about it, you don't need the if ;)

Reply:
smart